internal/service/vpc/resource_vpc_peering_connection.go

```diff
@@ -634,3 +635,11 @@ func validateVPCID(i interface{}, k string) (warnings []string, errors []error)
 
 	return warnings, errors
 }
+
+// ToNilOrStringPointer returns string pointer or nil
+func ToNilOrStringPointer(s string) *string {
```

Comment:
It looks we can reuse `OptionalStringPointer` instead of this function

Reply:
Yeah, have tested too. Unfortunately due to some unknown issues I can't provide a working test. I'll try to do that one more time. Function has been renamed.